integration-tests/initiateDataRequest.spec.ts

```diff
@@ -192,6 +198,61 @@ describe('Submit a PII request with approved ticket data', () => {
     })
   })
 
+  describe('valid requests for standard and glacier copy - analysis bucket empty', () => {
```

Comment:
Could you add a scenario for data in both tiers already in analysis bucket? Only one test case would do, as we don't have that for the mixed tier.

Reply:
sure

Reply:
i think this might just be a duplicate of the one we already have as once it's in the analysis bucket, any file is in the standard tier, regardless if it was originally stored in glacier. 
Also, if we move it a file from glaicer to the analysis bucket during the test, it won't be complete for running the test.

Reply:
We can't have both tiers in the analysis bucket. They are all converted to standard tier. So this test isn't relevant I would say!

Reply:
Plus the tier isn't relevant, the app works by comparing paths. It doesn't consider the data tier at all for this functionality.

Reply:
My thinking was to add data to the analysis bucket in the set up, with the audit bucket data a mix of standard and glacier. It is similar to the `valid requests - data present in analysis bucket` scenario, but this time the audit data would be in both tiers. I thought it would make for an interesting scenario to see if it the lambda tries to defrost or copy if that were the case.